eos/parameters/hadron-properties.yaml

```diff
@@ -22,6 +22,20 @@ groups:
           latex:   '$m_{\pi^\pm}$'
           unit:     'GeV'
           comments: "cf. [PDG-Live], December 2020"
+      'mass::eta' :
+          central: +0.547862
+          min:     +0.547845
+          max:     +0.547879
+          latex:   '$m_{\eta}$'
+          unit:     'GeV'
+          comments: "cf. [PDG-Live], December 2020"
+      'mass::eta_prime' :
```

Comment:
Why not ``"mass::eta'"``?

Reply:
I think I would like to keep `eta_prime` everywhere:

1. It is simpler to read than `"eta'"`
2. "'" is not allowed in the prefix part of qualified names
3. "eta'" is already used in the HQET form factors

What do you think?

Reply:
The readability part I understand. However, we have precendents for using ``eta'`` or even ``eta''`` in the HQET form factors, as you say. Ultimately, does not matter too much. Keep it as is. If it is en embuggerance later on, we  can still change it!